modules/@apostrophecms/image/ui/apos/components/AposMediaManager.vue

```diff
@@ -327,7 +348,8 @@ export default {
       });
     },
     async completeUploading (imgIds) {
-      this.uploading = false;
+      this.currentPage = 1;
```

Comment:
Why don't we always set `isLoading: true` when we load the media?

Reply:
Because the upload behavior is quite specific and was broken when hidding the upload block (it has a ref).